edge-modules/TestAnalyzer/Program.cs

```diff
@@ -1,27 +1,29 @@
 // Copyright (c) Microsoft. All rights reserved.
-namespace MessagesAnalyzer
+namespace TestAnalyzer
 {
     using System;
+    using System.Collections.Generic;
     using System.Linq;
     using System.Runtime.Loader;
     using System.Threading;
     using System.Threading.Tasks;
     using Microsoft.AspNetCore;
     using Microsoft.AspNetCore.Hosting;
     using Microsoft.Azure.Devices.Common;
-    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
     using Microsoft.Azure.EventHubs;
     using Microsoft.Extensions.Logging;
 
     class Program
     {
-        static readonly ILogger Log = Logger.Factory.CreateLogger<Program>();
+        static readonly ILogger Logger = ModuleUtil.CreateLogger(nameof(TestAnalyzer));
```

Comment:
```suggestion
        static readonly ILogger Logger = ModuleUtil.CreateLogger<TestAnalyzer>();
```

Reply:
ModuleUtil.CreateLogger does not support this syntax so I will keep nameof, unless you think I need to revert back to Logger.Factory.

Reply:
all good.